media/src/objects/Curve.svelte

```diff
@@ -465,7 +471,7 @@
 </script>
 
 <div class={'boxItem' + (selected ? ' selected' : '')} on:keydown>
-    <ObjHeader bind:hidden {onClose} {color} {onSelect}>
+    <ObjHeader bind:hidden {toggleHide} {onClose} {color} {onSelect} objHidden={!tube.visible}>
```

Comment:
It's kind of confusing to have both `hidden` and `objHidden` here. The `hidden` attribute really refers to minimizing the object form's window. Can we rename this to `minimized`? Then, we can use `hidden` in the place of `objHidden`.

Reply:
Changed `hidden` references referring to the minimize feature to `minimize`